### ISSUE TITLE
Local MySQL Database URL

### ISSUE THREAD

**Issue by dghubble** [kennethreitz/dj-database-url#19]
tl;dr; The MySQL URL string does not seem to accept parameters as expected.

In a Django project, this local database configuration:

    DATABASES = {
        'default': {
            'ENGINE': 'django.db.backends.mysql', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
            'NAME': 'example_db',        # Or path to database file if using sqlite3.
            'USER': 'myname',                   # Not used with sqlite3.
            'PASSWORD': 'mypass',            # Not used with sqlite3.
            'HOST': 'localhost',                  # Set to empty string for localhost. Not used with sqlite3.
            'PORT': '3306',                       # Set to empty string for default. Not used with sqlite3.
        }
    }

works just fine when I run 'python manage.py syncdb' and creates tables in local mysql db example_db nicely, so the user and permissions are set correctly.

I'd like to use dj_database_url instead:

    import dj_database_url
    # Uses DATABASE_URL environment variable if available. Otherwise, the default
    DATABASES['default'] =  dj_database_url.config(default='mysql://localhost:3306/example_db?username=myname&password=mypass')

I cannot run syncdb anymore then because I get an access denied error. The username and password are NOT being read from the MySQL string because the username Django tries is not 'myname' and it probably isn't reading the password correctly.

"Access denied for user 'some_other_username'@'localhost' (using password: NO)")

Searched around on SO and tried many different ways of writing the default string 'mysql://...' without much luck. I'm guessing the default string is malformed and I could use guidance.

**Comment by sudheesh001** [comment]
@kennethreitz This issue can be safely closed.
